src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -285,6 +285,11 @@ library BeaconChainProofs {
             "BeaconChainProofs.verifyWithdrawal: withdrawalIndex is too large"
         );
 
+        require(
+            withdrawalProof.historicalSummaryIndex < 2 ** HISTORICAL_SUMMARIES_TREE_HEIGHT,
+            "BeaconChainProofs.verifyWithdrawal: historicalSummaryIndex is too large"
+        );
+
```

Comment:
This is the other hexens finding, EIG 10.

Reply:
gotcha, so the issue here was being able to select an invalid `historicalSummaryIndex` ? Looks like test coverage was added too 👍